packages/analytics-js-integrations/src/integrations/CustomerIO/browser.js

```diff
@@ -34,17 +39,23 @@ class CustomerIO {
   }
 
   init() {
```

Comment:
can `dataUseInApp` be cleaned up from destination definition too ?

Reply:
yes, we can remove it but let's take it in separate pr as this one is only for logger changes